swap/src/network.rs

```diff
@@ -19,35 +17,3 @@ impl Executor for TokioExecutor {
         let _ = self.handle.spawn(future);
     }
 }
-
-#[derive(Debug, Clone, Copy, Eq, PartialEq)]
-pub struct Seed([u8; SEED_LENGTH]);
-
-impl Seed {
-    /// prefix "NETWORK" to the provided seed and apply sha256
-    pub fn new(seed: crate::seed::Seed) -> Self {
-        let mut engine = sha256::HashEngine::default();
-
-        engine.input(&seed.bytes());
-        engine.input(b"NETWORK");
```

Comment:
🔧 If we remove this, the default `PeerId` we set in the `swap_cli` will change.
I think we should do the change backwards compatible by adding the `NETWORK_` prefix when we derive the peer-id.